libraries/chain/market_evaluator.cpp

```diff
@@ -198,6 +198,8 @@ void_result call_order_update_evaluator::do_apply(const call_order_update_operat
    auto itr = call_idx.find( boost::make_tuple(o.funding_account, o.delta_debt.asset_id) );
    const call_order_object* call_obj = nullptr;
 
+   price old_collateralization;
```

Comment:
Perhaps make this an optional\<price> and use its truth value instead of is_null() further down?

Reply:
That's what I coded in the first place, however, `optional<>` will produce a warning on compile that's why I don't like it. It does provide a bit more performance though.

Reply:
Oh, it doesn't produce a warning here. Will make the change soon.